backend/history/postgresqlratingservice.py

```
@@ -0,0 +1,185 @@
+                port=self.port,
+                database=self.database,
+                user=self.username,
+                password=self.password,
+                min_size=self.pool_min_size,
+                max_size=self.pool_max_size
+            )
+            
+            # Ensure table exists
+            await self._ensure_table()
+            
+            return True, "PostgreSQL rating service initialized successfully"
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)